crypto/cipher/errors.go

```diff
@@ -0,0 +1,17 @@
+package cipher
+
+import (
+	"github.com/pkg/errors"
```

Comment:
Can you use the `import "errors"` this will mean the error does not have any stacktrace information. `"github.com/pkg/errors"` is incredibly useful but in this case we want to return a plain error

Reply:
done